### PR TITLE
Hide collected collections and show unobtained collection list

### PR DESCRIPTION
**Pull request by Grillo-del-mal**

### CODE REVIEW

**Comment by TehSeph** [comment]
Aww damn. I was gonna do this! :disappointed: 

Haven't tested, but looks good though. :+1:

**Comment by Excrulon** [comment]
Any chance you can make this into an ipf instead and commit it to the data folder along with committing the source where it currently is? I'm slowly phasing out the addon loader and all addons going forward should be packed in an ipf. I'm converting my current addons to that over time as well.

**Comment by Grillo-del-mal** [comment]
I'll do that tonight ;)

**Comment by Grillo-del-mal** [comment]
Implemented the ipf part using expcardcalculator implementation as reference.

Also implemented @TehSeph suggestion, now it looks great :+1:

**Comment by zckrs** [comment]
Is possible to change the background color of collection when one item is present in inventory?

File: addons/addonloader.lua [hunk 1]
```diff
@@ -19,6 +19,7 @@ dofile("../addons/monsterframes/monsterframes.lua");
 dofile("../addons/monstertracker/monstertracker.lua");
 dofile("../addons/showinvestedstatpoints/showinvestedstatpoints.lua");
 dofile("../addons/showpetstamina/showpetstamina.lua");
+dofile("../addons/hidecollected/hidecollected.lua");
```
Comment (Excrulon) [inline addons/addonloader.lua]: Don't think this is needed anymore, right?

**Comment by Grillo-del-mal** [comment]
https://github.com/MrJul/ToS-EnhancedCollection is better than this :+1:

**Pull request closed without merging**